README.md

```diff
@@ -54,23 +53,22 @@ The official Meteor resources page can be found [here](https://www.meteor.com/to
 
 *Where to start*
 
-* [Official Meteor tutorial](https://www.meteor.com/try)
-* [Discover Meteor Book](https://www.discovermeteor.com/)
```

Comment:
maybe you can update this link to this: http://www.discovermeteor.com/ ?

Reply:
This link doesn't seem to be working. Is it working on your end?

Reply:
yes, the `http` one, not the `https` one